pages/wg/outreach.md

```diff
@@ -75,6 +75,10 @@ a speaker series aimed at bringing representatives from funders (e.g., NSF, NIH)
 to speak to members of US-RSE. These funders will bring essential information
 about opportunities, strategies, and starting points for finding funding.
 
+- __Hal Finkel__ - Advanced Scientific Computing Research, Office of Science, US Department of Energy
```

Comment:
```suggestion
- __Hal Finkel__ - Advanced Scientific Computing Research, Office of Science, US Department of Energy: [video](https://youtu.be/vZYQJ5eOMls) & [slides](https://drive.google.com/file/d/1W9Oh50BMwHG6o1OEmlRXi_AaJYmMdMVn/view?usp=sharing)
```

Reply:
just trying to make this a bit more compact, with the idea that we will one day have lots of rows here.  If you accept this, you would also want to delete the next two lines